core/vm/contracts.go

```diff
@@ -1179,6 +1181,9 @@ func (c *l1sload) RequiredGas(input []byte) uint64 {
 }
 
 func (c *l1sload) Run(state StateDB, input []byte) ([]byte, error) {
+	log.Info("l1sload", "input", input)
```

Comment:
```suggestion
	log.Debug("l1sload", "input", input)
```

Reply:
Is it intentional to set `log.Info` here? btw, can print block number here for easier searching of the specific tx containing l1sload in other tools like scrollscan.